book/07-git-tools/sections/credentials.asc

```diff
@@ -1,29 +1,29 @@
 [[_credential_caching]]
-=== Credential Storage
+=== Kredensyal na ImbakanCredential Storage
 
 (((credentials)))
 (((git commands, credential)))
-If you use the SSH transport for connecting to remotes, it's possible for you to have a key without a passphrase, which allows you to securely transfer data without typing in your username and password.
-However, this isn't possible with the HTTP protocols – every connection needs a username and password.
-This gets even harder for systems with two-factor authentication, where the token you use for a password is randomly generated and unpronounceable.
-
-Fortunately, Git has a credentials system that can help with this.
-Git has a few options provided in the box:
-
-* The default is not to cache at all.
-  Every connection will prompt you for your username and password.
-* The ``cache'' mode keeps credentials in memory for a certain period of time.
-  None of the passwords are ever stored on disk, and they are purged from the cache after 15 minutes.
-* The ``store'' mode saves the credentials to a plain-text file on disk, and they never expire.
-  This means that until you change your password for the Git host, you won't ever have to type in your credentials again.
-  The downside of this approach is that your passwords are stored in cleartext in a plain file in your home directory.
-* If you're using a Mac, Git comes with an ``osxkeychain'' mode, which caches credentials in the secure keychain that's attached to your system account.
-  This method stores the credentials on disk, and they never expire, but they're encrypted with the same system that stores HTTPS certificates and Safari auto-fills.
-* If you're using Windows, you can install a helper called ``Git Credential Manager for Windows.''
-  This is similar to the ``osxkeychain'' helper described above, but uses the Windows Credential Store to control sensitive information.
-  It can be found at https://github.com/Microsoft/Git-Credential-Manager-for-Windows[].
-
-You can choose one of these methods by setting a Git configuration value:
+Kung ikaw ay gagamit ng SSH transport para sa pagkonekta ng mga remote, ito ay posible para sa iyo na magkaroon ng isang susi na walang isang passphrase, na kung saan ay nagpapahintulot sa iyo ng ligtas na paglipat ng datos na walang pag-type ng iyong username at password.
+Gayunpaman, ito ay hindi posible sa HTTP na mga protocol – sa bawat koneksyon ay nangangailangan ng isang username at password.
+Ito ay naging mas mahirap para sa mga sistema na may dalawang-factor sa pagpapatotohanan, na kung saan ang token na iyong ginamit para sa password ay randomly na nabuo at unpronounceable.
```

Comment:
- `randomly` to sapalaran
- `unpronounceable` to `hindi mabibigkas`

Reply:
done change